temporal/t.rast.to.vect/t.rast.to.vect.py

```diff
@@ -183,6 +183,7 @@ def main(options, flags):
                                    output="dummy", run_=False,
                                    finish_=False, flags=flags,
                                    type=method, overwrite=overwrite,
+                                   column=column,
```

Comment:
Well, I'm not sure. This is a call to `r.to.vect` which default for `column` parameter is *value*. Hence dunno if needed here. In the end the module converts each raster from a strds into a vector and registers them as stvds, what other thing would we store when converting a raster to a vector if not the value? All the examples in r.to.vect omit the column parameter

Reply:
In both cases, the documentation says it is _Name of attribute column to store value_. So according to the doc, it is a _name_, not _what_ should be be stored. (In theory, you could store, e.g., color, but that's not the case here.) Then it makes sense to pass it and since there is a default which is often good enough, it is not surprising that nobody complained.

Reply:
Yes, it puts *value* as default column name. Since there's no option to store any other attribute, I believe it's a reasonable default name. Of course, maybe better to have it explicitly there in case eventually `r.to.vect` allows to store something else and hence the name value would no longer apply